auslib/blobs/apprelease.py

```diff
@@ -118,6 +118,16 @@ def _getSpecificPatchXML(self, patchKey, patchType, patch, updateQuery, whitelis
         return '        <patch type="%s" URL="%s" hashFunction="%s" hashValue="%s" size="%s"/>' % \
             (patchType, url, self["hashFunction"], patch["hashValue"], patch["filesize"])
 
+    def validate(self, *args, **kwargs, ignore=False):
```

Comment:
Regarding the `ignore` parameter.. 
### Purpose
- To be used while testing, I noticed that the `hashValue` in most of the test cases is a single digit number and not a _dummy hash_, so during testing we will flip `ignore = True`